Automattic/wp-calypso
# Replace Subscriptions section available on Reader sidebar with Popular sites section

Related to #https://github.com/Automattic/loop/issues/233

## Proposed Changes

* In Reader recent screen replaced `ReaderListFollowedSites` component with `ReaderPopularSitesSidebar`
* Added `title` prop in `ReaderPopularSitesSidebar` component to make `title` implementation consistent.
* Removed the unused `ReaderListFollowedSites` component.

**Note:** The current implementation displays `10` popular sites without pagination. Let me know if it need any changes.

|Before|After|
-------|-----|
|![image](https://github.com/user-attachments/assets/30ae7fa8-efa6-4027-ad6a-e9cdb2a2595e)|![image](https://github.com/user-attachments/assets/5219245e-5a13-4f17-9bae-3a6b9e5242a2)|

## Why are these changes being made?


* New design requirements.

## Testing Instructions



* Navigate to `/read`.
* Make sure "Popular sites" on sidebar are working as expected.

## Pre-merge Checklist
